src/procedures/TransferReservationOwnership.ts

```diff
@@ -0,0 +1,59 @@
+import { Procedure } from './Procedure';
+import {
+  ProcedureType,
+  PolyTransactionTag,
+  TransferReservationOwnershipProcedureArgs,
+  ErrorCode,
+} from '../types';
+import { PolymathError } from '../PolymathError';
+import { SecurityTokenReservation } from '../entities';
+import { Factories } from '../Context';
+
+export class TransferReservationOwnership extends Procedure<
+  TransferReservationOwnershipProcedureArgs
+> {
+  public type = ProcedureType.TransferReservationOwnership;
+
+  public async prepareTransactions() {
+    const { newOwner, symbol } = this.args;
+    const {
+      contractWrappers: { securityTokenRegistry },
+      currentWallet,
+      factories,
+    } = this.context;
+
+    const { status, owner } = await securityTokenRegistry.getTickerDetails({
+      ticker: symbol,
+    });
+
+    const account = await currentWallet.address();
+
+    if (status) {
+      throw new PolymathError({
+        code: ErrorCode.ProcedureValidationError,
+        message: `You can not transfer a Security Token already launched`,
+      });
+    }
+
+    if (account === owner) {
```

Comment:
There should be two different checks here.

- One that compares `account` with `owner` (like this one) and throws an error saying "Only the reservation owner can transfer ownership to another wallet" if they're different
- One that comparse `owner` with `newOwner` and throws an error saying "New owner must be different from the current one to transfer ownership" if they're the same